webview/src/shared/components/dragDrop/DragDropContainer.tsx

```diff
@@ -100,7 +115,11 @@ export const DragDropContainer: React.FC<DragDropContainerProps> = ({
 
   useEffect(() => {
     cleanup()
-  }, [order])
+  }, [order, cleanup])
+
+  useLayoutEffect(() => {
+    onLayoutChange?.()
```

Comment:
This is going to be really helpful for [this](https://github.com/iterative/vscode-dvc/pull/2257/files#r955440333) as well